README.md

```diff
@@ -11,6 +11,8 @@ and using Python to solve problems or automate tasks.
 
 ## Getting Started
 
```

Comment:
you should be able to add the image file now, but will you make a new folder called "images" and move it there. Then add it in using markdown described here https://guides.github.com/features/mastering-markdown/